### ISSUE TITLE
Blank configuration save/load related...

### ISSUE THREAD

**Issue by Ketan-Desai** [cemdervis/SharpConfig#29]
Sir,

Blank configuration saved to file creates file with size 0 but loading same throws exception.

Technically, any configuration successfully saved must be loadable?

```
            var cfg = new Configuration();
            cfg.SaveToFile(@"d:\sample.ini");

            cfg = Configuration.LoadFromFile(@"d:\sample.ini");

```

**Comment by cemdervis** [comment]
Hey Ketan-Desai, you're correct. Let me see...

**Comment by cemdervis** [comment]
Fixed with version 1.5.6!
